home_page/implications/find_equation_id.js

```diff
@@ -18,12 +18,12 @@ function closePopup(element_id) {
     document.getElementById(element_id).style.display = 'none';
 }
 
-function tokenize(expr) {
-    return expr.replace(/\./g, '◇').replace(/\*/g, '◇').replace(/\(/g, ' ( ').replace(/\)/g, ' ) ').replace(/◇/g, ' ◇ ')
-        .split(' ')
-        .filter(token => token.trim() !== '');
-}
-
+//function tokenize(expr) {
```

Comment:
Ditto, delete this and the functions below also seem to be unused now so also delete them.